app/src/routes/[projectId]/+layout.ts

```diff
@@ -27,6 +28,7 @@ export async function load({ params, parent }) {
 	let project: Project | undefined = undefined;
 	try {
 		project = await projectService.getProject(projectId);
+		invoke('set_project_active', { id: projectId }).then((_r) => {});
```

Comment:
It explicitly doesn't await here to not block, but I don't know if there is any syntax that makes the intent clearer.

Reply:
Does this work? If it's returning a future then not awaiting it won't make it tick, will it?

Reply:
No it will, promises always execute immediately whether or not you await them.

Reply:
It is effective, and won't block the frontend. It is like `tokio::spawn()`, which is done for each and every promise in the frontend.

Reply:
Oh derp this is Typescript, not Rust. My bad :D